n64.mk

```diff
@@ -40,16 +41,21 @@ ASFLAGS+=-MMD    # automatic .d dependency generation
 %.z64: CFLAGS+=$(N64_CFLAGS)
 %.z64: ASFLAGS+=$(N64_ASFLAGS)
 %.z64: LDFLAGS+=$(N64_LDFLAGS)
-%.z64: $(BUILD_DIR)/%.elf
-	@echo "    [N64] $@"
-	$(N64_OBJCOPY) $< $<.bin -O binary
+%.z64: $(BUILD_DIR)/%.bin
+	@mkdir -p $(dir $@)
+	@echo "    [N64TOOL] $@"
```

Comment:
In general I think I'd prefer to hide the different tools both from beginner users and from the progress. They are just an internal implementation detail on how today libdragon does it with different tool. We could decide to merge everything into n64tool, or remove some parts (eg: checksum might not be needed in a OSS IPL3). I don't think it also gives a value to most builds having 3 different lines rather than one. For most users, this will just be a final step to convert the linker output to the ROM format, and that's probably enough. People can still look into this with V=1 if they want so we are not removing debugging power for instance. But in "default silent mode", I think one line is enough.